bismuth.spec

```diff
@@ -29,19 +30,29 @@ them via keyboard, just like in classical tiling window managers
 
 %files
 %license LICENSES/MIT.txt
+%{_prefix}/lib/debug%{_libdir}/qt5/plugins/org.kde.kdecoration2/bismuth_kdecoration.so-%{version}-%{release}.*
+%{_prefix}/lib/debug%{_libdir}/qt5/qml/org/kde/bismuth/core/libbismuth_core.so-%{version}-%{release}.*
```

Comment:
These lines should be removed, `rpmbuild` should automatically generate `debuginfo` packages and the main package shouldn't have any debug information.